vault/identity_store_entities.go

```diff
@@ -174,7 +182,7 @@ func (i *IdentityStore) pathEntityMergeID() framework.OperationFunc {
 			return nil, err
 		}
 
-		userErr, intErr := i.mergeEntity(ctx, txn, toEntity, fromEntityIDs, force, false, false, true)
+		userErr, intErr := i.mergeEntity(ctx, txn, toEntity, fromEntityIDs, conflictingAliasIDsToKeep, force, false, false, true, false)
```

Comment:
Rather than adding parameters to this function and setting them in the calling code directly, I think it would be more readable to wrap this function for the manual case and the automatic case, set the extra parameters as necessary for the internal merge call in the wrappers.